src/topology/channel.c

```diff
@@ -60,7 +60,7 @@ static const struct map_elem channel_map[] = {
 };
 
 
-static int lookup_channel(const char *c)
+int lookup_channel(const char *c)
```

Comment:
This first commit needs split up into several patches since it's doing multiple things and going to ALSA with commit messages.

Reply:
Sure, I am still refactoring these patches and working on improving the commit messages. There's about 3-4k lines of code added here and a lot modifed from the original just as a way to reuse them. Maybe I should split them into 2 PRs. Ones as the preparatory ones for TPLG2 and followed by the TPLG2 patches.